### PR TITLE
Ubuntu 20.04

### LINKED ISSUE #85: Stopwatch mode is still enabled on some servers/maps
Please leave a comment if you notice stopwatch mode on a particular server/map.

It's possible that this is only happening after empty server restart which would explain the pl_badwater and inconsistency in reports

### PR DESCRIPTION
**Pull request by JONTTM**
Update:
This PR should be merged because the servers are already running it. Future PRs will be tested on a specific test server. 

All servers have been returned to master branch, this pull request reflects what is currently on the servers.
In the future, I will have a dedicated server for testing new things like stopwatch mode to prevent some of the issues this has caused.

Closes #85 (by returning to master branch and reinstalling)

All servers except us3 are now running Ubuntu 20.04

Reasoning behind zsh:
I type `bash` every time I ssh into a server and I don't think @leighmacdonald uses the server consoles that much anyways. zsh will still be kept on the SF server because he owns it, but I would prefer bash on the others so I don't have to do this every time.

### CODE REVIEW

File: roles/tf2/files/50unattended-upgrades [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // from non-security sources (e.g. chromium). By allowing the release
```
Comment (PazerOP) [inline roles/tf2/files/50unattended-upgrades:4]: Apparently, this (generated?) file should not have been committed in the first place, and even if we did want to use the autoupgrade mechanism, its in the wrong role anyway.

File: roles/tf2/tasks/main.yml [hunk 2]
```diff
@@ -98,13 +98,6 @@
...
     dest: ~/serverfiles/tf/addons/sourcemod/configs/gbans.cfg
 
-- name: 20auto-upgrades
-  become: yes
-  copy:
-    src: 20auto-upgrades
-    dest: /etc/apt/apt.conf.d/20auto-upgrades
-    mode: '644'
-
 - name: 50unattended-upgrades
```
Comment (PazerOP) [inline roles/tf2/tasks/main.yml:101]: I assume this needs to be removed as well.

**Review by PazerOP — DISMISSED** [review]
See comments

**Comment by JONTTM** [comment]
@PazerOP Thanks for the review. I'll take a look on Thursday.

**Pull request merged**